x-pack/solutions/observability/plugins/observability_ai_assistant_app/server/rule_connector/index.ts

```diff
@@ -314,19 +314,13 @@ If available, include the link of the conversation at the end of your answer.`
       signal: new AbortController().signal,
       kibanaPublicUrl: (await resources.plugins.core.start()).http.basePath.publicBaseUrl,
       instructions: [backgroundInstruction],
+      systemMessage: getSystemMessageFromInstructions({
+        availableFunctionNames: functionClient.getFunctions().map((fn) => fn.definition.name),
+        applicationInstructions: functionClient.getInstructions(),
+        userInstructions: [],
+        adHocInstructions: functionClient.getAdhocInstructions(),
+      }),
```

Comment:
Do we really need to construct the system message here? It's also constructed inside `complete`. Notice how `instructions` is also part of the system message